page-visibility/prerender_call.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<title>Prerender test for Page Visibility API</title>
+<link rel="prerender" href="resources/prerender_target.html" pr="1.0">
```

Comment:
We can drop the `pr` attribute, we plan to remove that from the spec and no browser implements it.